### PR TITLE
Cge 31 my profile

### PR DESCRIPTION
**Pull request by Wolshebnik**
fixed bugs within the ticket
CGE-100;
CGE-102;
CGE-103;
CGE-104;
CGE-107;

### CODE REVIEW

File: src/main/resources/react/src/js/main.js [hunk 1]
```diff
@@ -46,48 +70,48 @@ document.addEventListener('DOMContentLoaded', function() {
...
-            document.querySelector('.modal.active').classList.remove('active');
-            document.querySelector('.overlay').classList.remove('active');
-        };
-    }, false);
-
-    overlay.addEventListener('click', function() {
-        document.querySelector('.modal.active').classList.remove('active');
-        this.classList.remove('active');
-    });
+      if (key == 27) {
```
Comment (rpkyrych) [inline src/main/resources/react/src/js/main.js:73]: Add comment that will describe the button that represents such key

File: src/main/resources/react/src/js/main.js [hunk 2]
```diff
@@ -46,48 +70,48 @@ document.addEventListener('DOMContentLoaded', function() {
...
-        }    
-    if(document.getElementById('tab4').checked) {        
-        document.getElementById('content1').style.display = 'none'; 
+        //document.getElementById('content5').style.display = 'none';
+        }
+    if(document.getElementById('tab4').checked) {
+        document.getElementById('content1').style.display = 'none';
         document.getElementById('content2').style.display = 'none';
         document.getElementById('content3').style.display = 'none';
         document.getElementById('content4').style.display = 'block';
```
Comment (rpkyrych) [inline src/main/resources/react/src/js/main.js:114]: The same code is duplicated 4 times here...

File: src/main/resources/react/src/pages/FormRegister/inputForForm/InputForForm.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const InputForForm = props => {
+const InputForForm = (props) => {
```
Comment (rpkyrych) [inline src/main/resources/react/src/pages/FormRegister/inputForForm/InputForForm.js:3]: :smile:

File: src/main/resources/react/src/pages/ProfilePage/active_profile/active_profile.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+const ActiveProfile = () => {
+  return (
+    <div className="active__profile">
+      <div className="active__profile__box">
+        <div className="active__profile__box-photo">
+          <img src={mommy} alt={"mother"} />
+        </div>
+        <div className="active__profile__box-data">
+          <p className="active__profile__box-data-name">
+            Альбина Задорожнaя,
```
Comment (rpkyrych) [inline src/main/resources/react/src/pages/ProfilePage/active_profile/active_profile.js:15]: Who's that?

File: src/main/resources/react/src/redux/actions/profile/addMe.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export const setAboutMe = (data) => ({
+  type: "ADD_ME_INPUT_VALUE",
+  payload: data,
+});
+
+export const setChildren = (data) => ({
+  type: "ADD_CHILDREN_INPUT_VALUE",
+  payload: data,
```
Comment (rpkyrych) [inline src/main/resources/react/src/redux/actions/profile/addMe.js:8]: Duplication of the logic.

**Review by rpkyrych — APPROVED** [review]
If you want to make some changes - go ahead, if not - it's your choice.

File: src/main/resources/react/src/pages/ProfilePage/active_profile/active_profile.js (hunk 4 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/pages/ProfilePage/active_profile/active_profile.js:15]: This is woman

Comment (rpkyrych) [reply src/main/resources/react/src/pages/ProfilePage/active_profile/active_profile.js:15]: No way?! :rofl:

**Review by Dariy98 — APPROVED** [review]

**Pull request merged**